src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -147,6 +147,44 @@ describe('ReactDOMComponent', function() {
       expect(console.error.argsForCall.length).toBe(2);
     });
 
+    it('should warn about styles with numeric string values for non-unitless properties', function() {
+      spyOn(console, 'error');
+
+      var div = document.createElement('div');
+      var One = React.createClass({
+        render: function() {
+          return this.props.inline ? <span style={{fontSize: '1'}} /> : <div style={{fontSize: '1'}} />;
+        }
```

Comment:
Linter requires trailing comma.  You can run the linter by executing `npm run lint`